src/pages/billing/Project.js

```diff
@@ -221,10 +224,42 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
 
+  const spendReportKey = 'spend report'
+  const maybeLoadTotalCost = reportErrorAndRethrow('Unable to retrieve spend report data')(async () => {
+    if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
+      setUpdatingTotalCost(true)
+      const endDate = new Date().toISOString().slice(0, 10)
+      const startDate = new Date((Date.now() - spendReportLengthInDays * 24 * 60 * 60 * 1000)).toISOString().slice(0, 10)
+      const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
+      const costFormatter = new Intl.NumberFormat(navigator.language, { style: 'currency', currency: spend.spendSummary.currency })
+      setTotalCost(costFormatter.format(spend.spendSummary.cost))
+      setUpdatingTotalCost(false)
+    }
+  })
+  maybeLoadTotalCost()
+
+  const CostCard = ({ title, amount, ...props }) => {
```

Comment:
The cost card can only handle 9 digits before it overflows onto the next line like this
![Screen Shot 2022-03-02 at 5 45 36 PM](https://user-images.githubusercontent.com/19961550/156462514-3b038fa6-786f-459f-b189-03b9c99e89d2.png)

This feels like a non-issue as our users aren't spending millions of dollars every 3 months as far as I know, but wanted to raise it in case you feel like I'm underestimating the issue

Reply:
I fixed this by allowing the card to grow if necessary. You see it grow, which isn't all that pretty, but the chances that any user actually has costs this high are extremely low.

![image](https://user-images.githubusercontent.com/484484/156586340-ee382bf8-23a2-46b7-8a79-82c182d5cfca.png)